### PR TITLE
Added configure section with note for Docker users

### PR DESCRIPTION
**Pull request by marcoliceti**

### CODE REVIEW

**Comment by harshavardhana** [comment]
Thanks for the PR @marcoliceti - reviewing now.

File: README.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ docker pull minio/mc:edge
 docker run minio/mc:edge ls play
 ```
 
+**Note:** These examples refer to the _play_ environment, a public Minio server for testing pruposes. If you want to try the same commands against a different environment, check [Configure](#configure).
```
Comment (harshavardhana) [inline README.md]: Typo s/pruposes/purposes/

File: README.md [hunk 2]
```diff
@@ -95,6 +97,30 @@ If you do not have a working Golang environment, please follow [How to install G
 go get -u github.com/minio/mc
 ```
 
+## Configure
+
+By default `mc` is configured to interact with the _play_ environment, a public Minio server. If you want to configure a different server:
```
Comment (harshavardhana) [inline README.md]: `If you want to configure a different S3 compatible server:`

File: README.md [hunk 3]
```diff
@@ -95,6 +97,30 @@ If you do not have a working Golang environment, please follow [How to install G
...
+
+```sh
+mc config host add <ALIAS> <YOUR-S3-ENDPOINT> <YOUR-ACCESS-KEY> <YOUR-SECRET-KEY>
+```
+
+You will be able to list buckets (i.e. `mc ls <ALIAS>`) or any other available operation.
+
```
Comment (harshavardhana) [inline README.md]: BTW this looks like a repeated mention.. i.e there are details about adding S3 and GCS below, we don't need to repeat the `#Configure` section. 

Perhaps you can move the `#Docker` section after `# Test your setup` section underneath.

File: README.md [hunk 4]
```diff
@@ -95,6 +97,30 @@ If you do not have a working Golang environment, please follow [How to install G
...
+### Endpoint and credentials
+
+At startup, Minio server provides its endpoint, access key and secret key.
```
Comment (harshavardhana) [inline README.md]: We don't need to talk about `Endpoint and credentials` from just Minio server point of view. `mc` is a generic tool to be used with all S3 compatible servers,  we can move this to `# Example - Minio Cloud Storage` section.

File: README.md (hunk 1 above)
Comment (marcoliceti) [inline README.md]: Fixed

File: README.md (hunk 2 above)
Comment (marcoliceti) [inline README.md]: Fixed

File: README.md (hunk 3 above)
Comment (marcoliceti) [inline README.md]: Removed duplicated info

File: README.md (hunk 4 above)
Comment (marcoliceti) [inline README.md]: Removed

File: README.md [hunk 5]
```diff
@@ -34,6 +34,14 @@ docker pull minio/mc:edge
 docker run minio/mc:edge ls play
 ```
 
+**Note:** These examples refer to the [_play_ environment](#test-your-setup). If you want to try the same commands against a different S3 compatible server, start the container this way:
```
Comment (NitishT) [inline README.md]: How about - Above examples run `mc` against Minio [_play_ environment](#test-your-setup) by default. To run `mc` against other S3 compatible servers, start the container this way:

File: README.md [hunk 6]
```diff
@@ -34,6 +34,14 @@ docker pull minio/mc:edge
...
+
+```sh
+docker run -it --entrypoint=/bin/sh minio/mc
+```
+
+then use the `mc config` command to [configure `mc`](#add-a-cloud-storage-service).
+
```
Comment (NitishT) [inline README.md]: then use the [`mc config`](#add-a-cloud-storage-service) command.

**Comment by marcoliceti** [comment]
@NitishT I updated the README with your suggestions.

**Comment by NitishT** [comment]
Thank you for the contribution @marcoliceti

**Pull request merged**